docs/use_cases/met_tool_wrapper/PlotDataPlane/PlotDataPlane_netcdf.py

```diff
@@ -0,0 +1,115 @@
+"""
+PlotDataPlane: GRIB1 Input
+==========================
+
+met_tool_wrapper/PlotDataPlane/PlotDataPlane_grib1.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Generate a postscript image to test if the input data can be read by the MET
+# tools.
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Input:** Sample GRIB1 file
+#
+# | **Location:** All of the input data required for this use case can be found in the met_test sample data tarball. Click here to the METplus releases page and download sample data for the appropriate release: https://github.com/dtcenter/METplus/releases
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See 'Running METplus' section for more information.
+#
+# | **Data Source:** Unknown
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case utilizes the METplus PlotDataPlane wrapper to generate a
+# command to run the MET tool PlotDataPlane if all required files are found.
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# PlotDataPlane is the only tool called in this example.
+# It processes the following run time:
+#
+# | **Valid:** 2007-03-30 0Z
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/met_tool_wrapper/PlotDataPlane/PlotDataPlane_grib1.conf
```

Comment:
Last comment for this config file; all references of Grib will need to change if this is indeed supposed to be a netCDF example. Otherwise, it's a duplicate of the previous config file example.